### PR TITLE
Fix button and text alignment in login, signup and forgot password page

### LINKED ISSUE #1141: Fix login/SignUp/Forgot password pages
**Describe the bug**
Improve alignment of buttons, text and social media handles for smaller displays. Remove the sketch on forgot password page for smaller displays.

**Screenshots**
image
image
image

### PR DESCRIPTION
**Pull request by rishipurwar1**
Fixes #1141 

#### Describe the changes you have made in this PR -
1. I remove sketch from forgot password page
**Before**
beta circuitverse org_users_password_new(Galaxy S5)

**After**
127 0 0 1_8080_users_password_new(Galaxy S5)

2. FIx button and text alignment in login and signup page
**Before**
beta circuitverse org_users_sign_in(Galaxy S5)

**After**

127 0 0 1_8080_users_sign_up(Galaxy S5) (1)
127 0 0 1_8080_users_sign_in(Galaxy S5)

### CODE REVIEW

**Comment by rishipurwar1** [comment]
Please check @aayush-05 @nitin10s @tachyons

**Comment by aayush-05** [comment]
Don't change padding for all screens, use media queries. Also attach screenshots for pc.

**Comment by rishipurwar1** [comment]
Ok, I'll attach a screenshot for pc. and use media query for padding. I want to know that changes made by me is it OK or not. @aayush-05

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -25,3 +25,7 @@
   float: none;
   margin: 0 auto;
 }
+
```
Comment (tachyons) [inline app/assets/stylesheets/application.scss]: Move to appropriate stylesheet

**Comment by nitin10s** [comment]
@rishipurwar1

**Comment by rishipurwar1** [comment]
Where should I move this code ?
@nitin10s @tachyons

**Comment by aayush-05** [comment]
Move it to signup.scss @rishipurwar1 .Also add the web screenshots in the description too

**Comment by aayush-05** [comment]
@rishipurwar1 are you still working on this ?

**Comment by nitin10s** [comment]
Don't actually, we'll be redesigning the login page as per last discussion.

**Comment by aayush-05** [comment]
Shall we close this @nitin10s ?

**Comment by nitin10s** [comment]
Closing, this we'll be changing the login system

**Pull request closed without merging**